README.md

```diff
@@ -48,6 +48,19 @@ end
 
 Use the batch execution strategy with your schema
 
+- When running `graphql` gem versions `>= 1.3.0`:
+
+```ruby
+MySchema = GraphQL::Schema.define do
+  query MyQueryType
+  
+  lazy_resolve(Promise, :sync)
+  instrument(:query, GraphQL::Batch::Setup)
+end
+```
+
+- When running `graphql` gem versions `< 1.3.0`:
```

Comment:
I would prefer to just recommend using lazy_resolve and deprecating the old way of doing it.

I have opened https://github.com/Shopify/graphql-batch/pull/42 to deprecate the old way, although I think I will need to also add a warning for when an old version of the graphql gem is used.